fetch/fetch.d.ts

```diff
@@ -0,0 +1,89 @@
+﻿// Type definitions for fetch API
+// Project: https://github.com/github/fetch
+// Definitions by: Ryan Graham <https://github.com/ryan-codingintrigue>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../typescript-services/typescriptServices.d.ts" />
```

Comment:
The `typescriptServices`? Needed as `Headers` is iterable and therefore implements `TypeScript.Iterator<string>`. Unless I'm using that wrong...